src/dtls.c

```diff
@@ -1384,7 +1372,42 @@ int wolfSSL_dtls_cid_get_tx(WOLFSSL* ssl, unsigned char* buf,
     return DtlsCidGet(ssl, buf, bufferSz, 0);
 }
 
+int wolfSSL_dtls_cid_max_size(void)
+{
+    return DTLS_CID_MAX_SIZE;
```

Comment:
Will this value change?
Just wondering why this function exists.

Reply:
The value can change at compile-time only. The reason for the function is that the information is useful for the application code.